LondonBritishLibrary/orient/BLorient613.xml

```diff
@@ -0,0 +1,334 @@
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+</msItem>
+
+</msItem>
+
+<msItem xml:id="ms_i2">
+<locus from="126v"/>
+<title xml:lang="en" type="complete" ref="LIT2722RepCh6">A hymn to St George</title>
+<incipit xml:lang="gez">መልክዓ፡ ሥቃይ፡ ሥዑላነ፡ መጽሔተ፡ ሥጋከ፡ አርአየ። እመ፡ አመ፡ ትቤ፡ ላዕሌከ፡ እስከ፡ ፈጸምከ፡ ሥቃየ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i3">
+<locus from="141r"/>
+<title type="complete" ref="LIT3942ArdeetCh"/>
+<incipit xml:lang="gez">በስመ፡ አብ፡ <gap reason="omitted" resp="PRS10171Wright"/>ነገር፡ በእንተ፡ አርድእት፡ ዘከመ፡ ነገሮሙ፡ ኢየሱስ፡ በዘይድኅኑ፡ እምኵሉ፡ ኃጢአት፡ ወለክሙኒ፡
+ንነግረክሙ፡ ንሕነ፡ በዘትድኅኑ፡ እምኵሉ፡ ኃጢአት፤ ወበዝ፡ ድኅኑ፡ እምኵሉ፡ እኩይ፡ ነገር፡ ወቅስት፤ እጓለ፡ እምሕያው፡ ወበዝ፡ ይድኅኑ፡ እምኵሉ፡ ፀራዊ፡ ወኅምዘ፡ አርዌ፡ ምድር፡ ወእምኵሉ፡ ዘቦቱ፡ ኅምዘ፡ ዘይቀትል፡ ነፍስ፡
+ወሥጋ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i4">
+<locus from="177v" to="181r"/>
+<locus from="2r" to="3r"/>
+<title xml:lang="en" type="complete" ref="LIT2981RepCh262">A hymn to the Holy Trinity</title>
+<incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ ጥሙቅ። እብል፡ አንሰ፡ ገባእተ፡ ሥላሴ፡ እምርሑቅ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+</msContents>
+
+<physDesc>
+
+  <objectDesc form="Codex">
+      <supportDesc>
+          <support>
+              <material key="parchment"/>
+          </support>
+          <extent>
+              <measure unit="leaf">181</measure>
+            <dimensions type="outer" unit="in">
+<height>7.375</height>
+<width>6</width>
+</dimensions>
+
+            </extent></supportDesc>
+
+            <layoutDesc>
+                <layout columns="2" writtenLines="17">
+                </layout>
+            </layoutDesc>
+
+</objectDesc>
+
+<handDesc>
+    <handNote script="Ethiopic" xml:id="h1">
+        <desc>Written in a fine hand of the earlier half of the 18th Century.</desc>
+        <date notBefore="1700" notAfter="1750" resp="PRS10171Wright"/>
+    </handNote>
+</handDesc>
+
+<additions>
+<list>
+
+<item xml:id="a1">
+<locus target="#1v"/>
+<desc type="ProtectivePrayer"><q xml:lang="gez">በእንተ፡ ሕማመ፡ ዐይን፡ ወርእስ።</q></desc>
+</item>
+
+  <item xml:id="a2">
+  <locus target="#4r"/>
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+            <placeName ref="INS0101MadhaneAlam"/>.</desc>
+        <q xml:lang="gez">ድርሳነ፡ ሚካኤል፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+</item>
+
+<item xml:id="a3">
+<locus target="#11v"/>
+<desc type="ProtectivePrayer">According to the note the original owner of the manuscript was <persName ref="PRS13401GabraIyasus">Gabra ʾIyasus</persName>.</desc>
+<q xml:lang="gez">ወኢይመውኦ፡ ጸላዒሁ፡ ወኢይሬኢ፡ ኵነኔ፡ አመ፡ ዕለተ፡ ሞቱ፡ ወይባልሖ፡ ቅዱስ፡ ሚካኤል፡ ለፍቁሩ፡ ገብረ፡ ኢየሱስ።</q>
+</item>
+
+<item xml:id="a4">
+<locus target="#4r"/>
+<desc type="OwnershipNote">A later owner was dabtarā <persName ref="PRS13400Estifanos">ʾƎsṭifānos</persName>.</desc>
+<q xml:lang="gez">ዝድርሳን፡ ዘደብተራ፡ እስጢፋኖስ፡</q>
+</item>
```

Comment:
please, add seg

Reply:
But this is a full text of this note, it is not a beginning. I thought that we use <seg> only when we're qutoing a fragment.